src/timelines/taskBlock.js

```diff
@@ -5,14 +5,8 @@ const taskBlock = (blockSettings) => {
   // initialize block
 	const startingOpts = generateStartingOpts(blockSettings)
 
-  const blockDetails = {
-	  block_earnings: 0.0,
-		optimal_earnings: 0.0,
-		continue_block: true
-	}
-
 	// timeline = loop through trials
```

Comment:
why not collect the overall earnings anymore?

Reply:
I don't think it was ever used anywhere.